src/models/platform.ts

```diff
@@ -232,7 +236,9 @@ export class PlatformInformation {
 
 	private static getWindowsArchitecture(): Promise<string> {
 		return new Promise<string>((resolve, reject) => {
-			if (process.env.PROCESSOR_ARCHITECTURE === 'x86' && process.env.PROCESSOR_ARCHITEW6432 === undefined) {
+			if (process.env.PROCESSOR_ARCHITECTURE === 'ARM64') {
```

Comment:
This doesn't look like it actuallly needs to return a promise

Reply:
the method itself doesn't have to be, but the place it is called also needs to be modified, so I'd leave it as is.
![image](https://user-images.githubusercontent.com/13777222/224794024-3a724f69-6179-4296-9854-6f8c868ab43c.png)